Makefile

```diff
@@ -145,14 +145,14 @@ ifeq ($(TAG),devel)
 	@$(BUILDER) pull clearlinux:latest
 endif
 
-images = $(shell ls build/docker/*.Dockerfile | sed 's/.*\/\(.\+\)\.Dockerfile/\1/')
+images = $(shell basename -a build/docker/*.Dockerfile | sed -r 's/(.+)\.Dockerfile/\1/')
```

Comment:
to get rid of `sed` completely we could

```
basename -a build/docker/*.Dockerfile | cut -d . -f 1
```

wdyt?

Reply:
To get rid of piping completely we could ```basename -s .Dockerfile -a build/docker/*.Dockerfile```